uber/config.py

```diff
@@ -106,18 +106,21 @@ def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
+            for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
+                if (dt or sa.localized_now()) >= day:
+                    price = bumped_price
+                    # If we set a price during the event, it should be used regardless of badge sales
+                    if c.EPOCH >= day >= c.ESCHATON:
+                        return price
+
             if dt or c.HARDCORE_OPTIMIZATIONS_ENABLED:
```

Comment:
No, we still want to check for `if dt`.

The way is works is that if a datetime is passed to this function, we assume that the badge whose price is being checked is not new (e.g., they're passing in the badge's `registered` date). We want to skip doing the badges sold calculation in that case. We relied on this for when we retroactively updated people's overridden price for groups. It's less important now, but it's still a useful optimization. I think.

You're kinda right about not needing to set dt though, so this could still become a one-liner. I'll do that.